recipe/meta.yaml

```diff
@@ -22,51 +22,48 @@ source:
     - 0001-shobjidl-Fix-compile-guard-around-SHARDAPPIDINFOLINK.patch  # [win]
 
 build:
-  number: 7
-  skip: True  # [osx or win]
+  number: 8
+  skip: True  # [ win]
   detect_binary_files_with_prefix: true
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and (py35 or py36)]
 
 requirements:
   build:
-    - python  # [win]
-    - python 2.7*  # [unix]
+    - perl
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - curl  # [win]
+    - python   3.6  # [osx]
+    - python   2.7  # [linux]
```

Comment:
Just curious, what is going on here? Why do we need two different version of Python for the two different OSes?